tests/testutils.go

```diff
@@ -37,18 +37,24 @@ func ExecAtlasSQLScript(sqlFilePath string) {
 	ExecSQLScript(sqlFilePath, -1)
 }
 
-func ExecSQLScript(sqlFilePath string, sourceId int) {
+func ExecSQLScript(sqlFilePath string, sourceId int) string {
 	log.Printf("Loading %s...", sqlFilePath)
-
+	errString := ""
 	path, err := filepath.Abs(sqlFilePath)
 	if err != nil {
 		panic(err)
 	}
 	fileContents, err2 := os.ReadFile(path)
 	if err2 != nil {
-		panic(err)
+		panic(err2)
 	}
-	ExecSQLString(string(fileContents), sourceId)
+	result := ExecSQLString(string(fileContents), sourceId)
+	if result.Error != nil && len(result.Error.Error()) > 0 {
+		errString = result.Error.Error()
+		fmt.Printf("Errors: %s\n", errString)
```

Comment:
```suggestion
		fmt.Printf("Errors: %s\n", errString)
		panic(errString)
```
I think it makes sense to panic here as well after printing the error